static/src/javascripts/projects/commercial/modules/dfp/dfp-epic-slot.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import timeout from 'lib/timeout';
+
+import { addSlot } from 'commercial/modules/dfp/add-slot';
+import { createSlots } from 'commercial/modules/dfp/create-slots';
+import { awaitGoogletagCmd } from 'commercial/modules/dfp/googletag-cmd';
```

Comment:
There is a command queue on the page until GoogleTag has finished initialising. You should be able to push to that, rather than awaiting?